modules/porous_flow/test/tests/sinks/PorousFlowPiecewiseLinearSink_BC_eg1.i

```diff
@@ -0,0 +1,131 @@
+## This is an example input file showing how to set a Type I BC with PorousFlowPiecewiseLinearSink
```

Comment:
Please make this a proper test by including it in the "test/tests/sinks/tests" file, along with a gold file.  This prevents it from getting outdated at some stage.  I've noted some things you can probably remove from this test so that it is minimised.

Could you please change "Type I BC" to "Type I (Dirichlet) BC"

Reply:
And include a slightly longer description of the test that includes a description of why the CSV file is correct

Reply:
Added description, simplified input file, made sure output fluxes match an analytical solution, and ensured that the pressure at the boundary is set correctly. Also, I added a gold file and modified the test file so that the gold file is compared with ./run_tests.